test/e2e/operator_test.go

```diff
@@ -25,13 +24,17 @@ import (
 	"testing"
 	"time"
 
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"k8s.io/client-go/util/retry"
```

Comment:
Please group `"k8s.io/client-go/util/retry"` with the other `k8s.io/client-go` imports, and please group `"github.com/aws/aws-sdk-go/service/ec2"` with `"github.com/aws/aws-sdk-go/aws/endpoints"`.

Reply:
Thanks, updated and grouped.